packages/former/src/components/FormNodeProps.vue

```diff
@@ -2,9 +2,11 @@
   <div v-if="selectedNode" class="flex flex-col gap-4 w-full">
     <div class="flex w-full justify-between">
       <span class="text-lg">{{ selectedNodeType?.label }}{{ selectedNode.name ? ` - ${selectedNode.name}` : '' }}</span>
-      <Button @click="selectedNode = undefined">
-        x
-      </Button>
+      <slot name="unselect-button" :unselect="unselectComponent">
```

Comment:
@mariusheine Slot works as well. I actually thought of using something like `const myComponents = { Button: props.myComponents.Button || SampleButton }` previously.

Reply:
@anbraten Yeah I was thinking of such thing as well but decided against it, because with your solution the button would then popup in the FormAdd component.

Reply:
@anbraten Ahh you wanted to introduce a new prop. That is also nice... Less verbose probably. I will try it out

Reply:
@anbraten With slots it is more flexible and headless in my opinion. Because when the component is a prop I also have to add the texts as props. So I prefer the slot approach

Reply:
Works fine. Requires you to override quite a few places to adjust all buttons.